test/OpenTelemetry.Exporter.OpenTelemetryProtocol.FuzzTests/Generators.cs

```diff
@@ -0,0 +1,236 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using FsCheck;
+using FsCheck.Fluent;
+using OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation;
+using OpenTelemetry.Logs;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Resources;
+
+namespace OpenTelemetry.Exporter.OpenTelemetryProtocol.FuzzTests;
+
+internal static class Generators
+{
+    public static readonly ActivitySource TestActivitySource = new("Fuzz.ActivitySource", "1.0.0");
+
+    private static readonly Gen<ActivityStatusCode> ActivityStatusCodes = Gen.Elements(
+    [
+        ActivityStatusCode.Unset,
+        ActivityStatusCode.Ok,
+        ActivityStatusCode.Error,
+    ]);
```

Comment:
Candidate for separate PR:
```suggestion
    private static readonly Gen<ActivityStatusCode> ActivityStatusCodes = Gen.Elements(
    [
        ActivityStatusCode.Unset,
        ActivityStatusCode.Ok,
        ActivityStatusCode.Error,
        (ActivityStatusCode)int.MaxValue,
    ]);
```